src/business-logic/services/tv2-inews-ingest-service.ts

```diff
@@ -0,0 +1,12 @@
+import { IngestService } from './interfaces/ingest-service'
+import {HttpRequestParameters, HttpService} from './interfaces/http-service'
+
+export class Tv2InewsIngestService implements IngestService {
```

Comment:
Should the name not be `Tv2INewsIngestService`?

Reply:
We spell "iNews" with lowercase `i` and uppercase `N`, so yeah, it should technically be `Tv2INewsIngestService`.

Reply:
I have changed the naming, good catch :-)